frontend/public/components/nav/admin-nav.tsx

```diff
@@ -43,9 +43,14 @@ const monitoringAlertsStartsWith = [
   'monitoring/alerts',
   'monitoring/alertrules',
   'monitoring/silences',
+];
+const clusterSettingsStartsWith = [
+  'settings/cluster',
+  'settings/idp',
+  'config.openshift.io',
+  'monitoring/alertmanagerconfig',
```

Comment:
I would expect the `Alerting` nav item to be selected, not `Cluster Settings`.

Reply:
Does this mean we would show all 4 tabs/pages under Alerting? (Alerts, Silences, Configuration, YAML)?  This would be different from current design cc @cshinn

Reply:
Sorry, I missed that this moved to a separate page